packages/styles/src/elements/segmented-button.scss

```diff
@@ -0,0 +1,195 @@
+@use '../functions/tokens';
+@use '../tokens/components';
+@use '../mixins/utilities';
+@use '../core' as post;
+
+tokens.$default-map: components.$post-segmented-button;
+
+$post-segmented-button-max-count: 8;
+
+.segmented-button-container {
+  container-name: post-segmented-container;
+  container-type: inline-size;
+}
```

Comment:
Is it possible to specify this on the `<fieldset>` in order to get rid of the wrapping `<div>`?